### ISSUE TITLE
highlight find freeze on / search if a text input field has the same searched word

### ISSUE THREAD

**Issue by RFDAJE** [pentadactyl/pentadactyl#9]
This is a long standing bug, I already spend hours on it, but found no solution yet. My temporary solution is to disable highlight feature. see https://github.com/RFDAJE/pentadactyl/commit/ee69521b25313fca9993417f741333ea792d4aa5

upstream bug reports:
* https://github.com/5digits/dactyl/issues/206
* https://github.com/madand/pentadactyl-pm/issues/2

**Comment by thegala** [comment]
Did you find anything?

**Comment by RFDAJE** [comment]
> Did you find anything?

not so much.  it seems, the problem is not in `highlight()` function but searching text function `find()` or even deeper, during the debug session I found out, if searched string is in an input text field, the searching results iteration stuck at input field or in between of first found text and input text field, then browser freeze. if I kept track of them, and force a break in `highlight()` loop, only those string before text input will be highlighted, the rest of the found strings will be left out.

I have to spend more time to understand how finder.jsm works, but it is hard for me at the moment to get more spare time, so disable the feature as a easy work around.
